src/v0/destinations/active_campaign/util.js

```diff
@@ -1,31 +1,25 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
 const {
-  nodeSysErrorToStatus,
   getDynamicErrorType,
+  processAxiosResponse,
 } = require('../../../adapters/utils/networkUtils');
 const tags = require('../../util/tags');
 
 const errorHandler = (err, message) => {
-  if (err.response) {
-    throw new NetworkError(
-      `${message} (${err.response?.statusText},${JSON.stringify(err.response?.data)})`,
-      err.status,
-      {
-        [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(err.status),
-      },
-      err,
-    );
-  } else {
-    const httpError = nodeSysErrorToStatus(err.code);
-    throw new NetworkError(
-      `${message} ${httpError.message}`,
-      httpError.status,
-      {
-        [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(httpError.status),
-      },
-      err,
-    );
+  const {response, status} = processAxiosResponse(err);
+  const stringifiedResponse = JSON.stringify(response);
+  let msg = `${message} ${stringifiedResponse}`;
+  if (response) {
+    msg = `${message} (${err.response?.statusText},${stringifiedResponse})`;
   }
+  throw new NetworkError(
+    msg,
+    status,
+    {
+      [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(status),
+    },
+    response?.data,
```

Comment:
Maybe we should use `err` instead like it was before? It gives us more context of the error.

Reply:
The `response` object is derived from `err` by using `processAxiosResponse`. The object `response` indicates **response body**. We would have to change from `response?.data` it to `response`

Reply:
Yes, let's do it.

Reply:
Done